doc/02-Installation.md

```diff
@@ -13,10 +13,10 @@ chapter.
 ## Installing Requirements <a id="installing-requirements"></a>
 
 * A web server, e.g. Apache or nginx
-* PHP >= 5.3.0 w/ gettext, intl, mbstring and OpenSSL support
+* PHP >= 5.3.0 with gettext, intl, mbstring and OpenSSL support
 * Default time zone configured for PHP in the php.ini file
 * LDAP PHP library when using Active Directory or LDAP for authentication
-* Icinga 2.x w/ IDO feature enabled or Icinga 1.x w/ IDO
+* Icinga 2.x with the IDO feature enabled or Icinga 1.x w/ IDO
```

Comment:
Line 19:
Maybe better: Icinga 2.x or 1.x with the IDO feature enabled

Reply:
I'll refactor the chapter a bit, thanks for the pointer. I've started to remove references to 1.x in this commit in other locations already.